app/rtc_compare.py

```diff
@@ -186,7 +195,9 @@ def get_list_dataset_attrs_keys(hdf_obj_1: h5py.Group,
 
 
 def compare_hdf5_elements(hdf5_obj_1, hdf5_obj_2, str_key, is_attr=False,
-                          id_key=None, total_key=None, print_passed_element=True):
+                          id_key=None, total_key=None,
+                          print_passed_element=True,
+                          list_exclude: list=[]):
```

Comment:
Please, avoiding setting `list_exclude` default to `[]` because it's a mutable argument:
https://docs.python-guide.org/writing/gotchas/
Set it to `None` instead.

Reply:
Good point. Thanks for the suggestion @gshiroma!